src/wfcomman.c

```diff
@@ -1665,52 +1665,26 @@ AppCommandProc(register DWORD id)
       //
       WAITNET();
 
-      if (lpfnShareStop) {
-
-         if (ret = ShareCreate(hwndFrame)) {
-
-            //
-            // Report error
-            // ret must the error code
-            //
-            goto DealWithNetError;
-
-         } else {
+      if (lpfnShowShareFolderUI) {
+         LPTSTR szDir;
+         BOOL bDir = FALSE;
+         szDir = GetSelection(1 | 4 | 16, &bDir);
+         HRESULT hr = ShowShareFolderUI(hwndFrame, szDir);

```

Comment:
Please avoid C99 variable declarations.

(I know, I'm not trying to comment on the quality of the language construct.  It's just that the newest compiler that targets Windows 2000 doesn't support them, so I end up going through and replacing them all with C89.  This isn't really because Windows 2000 is an interesting target, it's because when somebody points out unexpected behavior I'm going through old releases to try to figure out what changed, when, and why.)